escher/src/text_area.cpp

```diff
@@ -513,21 +513,27 @@ void TextArea::ContentView::drawRect(KDContext * ctx, KDRect rect) const {
   }
 }
 
-void TextArea::ContentView::drawStringAt(KDContext * ctx, int line, int column, const char * text, int length, KDColor textColor, KDColor backgroundColor, const char * selectionStart, const char * selectionEnd, KDColor backgroundHighlightColor) const {
+void TextArea::ContentView::drawStringAt(KDContext * ctx, int line, int column, const char * text, int length, KDColor textColor, KDColor backgroundColor, const char * selectionStart, const char * selectionEnd, KDColor backgroundHighlightColor, bool isItalic) const {
   if (length < 0) {
     return;
-  }
-  KDSize glyphSize = m_font->glyphSize();
-
+  } 
+  const KDFont * ItalicFont = (m_font == KDFont::LargeFont) ?  KDFont::ItalicLargeFont : KDFont::ItalicSmallFont;
+  const KDFont * usedFont = isItalic ? ItalicFont : m_font;
```

Comment:
It's is not very clean, maybe a thing like :

```suggestion
  const KDFont * usedFont = m_font;
  if (isItalic) {
    usedFont = m_font->toItalic();
  }
```
with `toItalic()` a new method that can decline a font in its italic version. With this type of implementation, if one day we add a MediumFont, the code will still works.